src/applications/personalization/profile360/tests/util/index.unit.spec.js

```diff
@@ -7,36 +7,37 @@ import {
 } from '../../util';
 
 describe('profile utils', () => {
-  const defaultDataObject = {
```

Comment:
I moved this down a level and also added a helper to generate them because they are all so similar.